modules/oe_translation_epoetry/tests/src/FunctionalJavascript/EpoetryTranslationTest.php

```diff
@@ -2555,6 +2555,51 @@ public function testAutomatedRequestStatusUpdate(): void {
     }
   }
 
+  /**
+   * Tests that warning message is shown for requests with a weekend deadline.
+   */
+  public function testWeekendDeadlineWarning(): void {
```

Comment:
I'm sorry for all this extra work but for this tiny assertion you can just add to the `testEpoetrySingleTranslationFlow` where we also assert things like the mandatory language and the max length on the message. So, you can just add a few lines there instead of having an entire test running just for this.